validphys2/src/validphys/pseudodata.py

```diff
@@ -287,7 +287,10 @@ def level0_commondata_wc(data, fakepdf):
 
 
 def make_level1_data(
-    data, level0_commondata_wc, filterseed, experiments_index, sep_mult
+    data, level0_commondata_wc, filterseed, experiments_index,
+    sep_mult, MULT = False, ADD = False, CORR = False, UNCORR = False, SPECIAL=False,
```

Comment:
The way I read this function, the data should be already processed before calling this function, so I would remove these arguments and just process the data before calling `make_level1_data` since these are arguments that are only active when `type1_inconsistency` is set to `True`.